android/build.gradle

```diff
@@ -12,27 +12,17 @@ buildscript {
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-
-boolean dash = false, hls = false, smoothstreaming = false
-
-File file = file('../../../track-player.json')
-if(file.exists()) {
-    def json = new JsonSlurper().parseText(file.text)
-    dash = json.dash ?: dash
-    hls = json.hls ?: hls
-    smoothstreaming = json.smoothstreaming ?: smoothstreaming
-}
-
-def safeExtGet(prop, fallback) {
-    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
-}
+//
+//def safeExtGet(prop, fallback) {
```

Comment:
I would restore this and continue using it below.

Reply:
Here's another lib as a reference: https://github.com/react-native-video/react-native-video/blob/master/android/build.gradle